### ISSUE TITLE
Installing from head breaks against github.com/go-openapi/spec

### ISSUE THREAD

**Issue by pwittrock** [go-openapi/analysis#14]
```go
vendor/github.com/go-openapi/analysis/flatten.go:27: undefined: spec.ExpandOptions
vendor/github.com/go-openapi/analysis/flatten.go:28: undefined: spec.ExpandOptions
vendor/github.com/go-openapi/analysis/flatten.go:46: too many arguments in call to spec.ExpandSpec
	have (*spec.Swagger, <T>)
	want (*spec.Swagger)
vendor/github.com/go-openapi/analysis/flatten.go:155: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:406: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:443: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:455: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:539: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:543: undefined: spec.ResolveRefWithBase
vendor/github.com/go-openapi/analysis/flatten.go:550: undefined: spec.Debug
vendor/github.com/go-openapi/analysis/flatten.go:550: too many errors
```

**Comment by casualjim** [comment]
did you update the spec repo too?

**Comment by afritzler** [comment]
I just ran into the same problem here. Any updates on this one?

**Comment by casualjim** [comment]
the solution is you have to update the go-openapi libraries in your vendor.

**Comment by afritzler** [comment]
@casualjim the libs in my vendor folder are up to date. :-( Using glide for vendoring. Any idea what else I could do?

**Comment by casualjim** [comment]
can you set the version in glide for go-openapi/spec to master?
perhaps you need a `glide cc`

I use swagger from master with those packages in my project at work with glide as well.

in my glide.lock I have the following entry for go-openapi/spec
```yaml
- name: github.com/go-openapi/spec
  version: 9cf697d99bf904e29363382a2e8c23dde5c8d377
```

That commit is the current HEAD of go-openapi/spec

I have these guys in my glide.yaml

```yaml
- package: github.com/go-openapi/errors
- package: github.com/go-openapi/loads
- package: github.com/go-openapi/runtime
  subpackages:
  - client
  - flagext
  - middleware
- package: github.com/go-openapi/spec
- package: github.com/go-openapi/strfmt
- package: github.com/go-openapi/swag
- package: github.com/go-openapi/validate
```

**Comment by advanderveer** [comment]
I ran into the same issue. For unkown reason glide doesn't simply set dependencies at master. I had to manually set the commits which is a poor experience. Glide has never disappointed me so I am actually inclined to say that is isn't a bug in there part but unfortunately I have no time to find out for sure. The go-swagger project sure has a lot of dependencies though, al the way to mgo it seems...
